### PR TITLE
Optimize getAsset by calling evaluate rather than submit. 

### PR DESCRIPTION
**Pull request by rykal-nate**
### Checklist:

- [x] Read the [Code of Conduct](https://github.com/appsody/website/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md).

- [x] Followed the [commit message guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).

- [x] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md#stack-structure).

### Modifying an existing stack:

- [x] Updated the stack version in `stack.yaml`

### Contributing a new stack:

- Describe how application dependencies are managed:

- Explain how Appsody file watcher is utilized:

- Describe other Appsody environment variables defined by the stack image:

- Describe any limitations and known issues:

### CODE REVIEW

File: incubator/blockchain-client-openliberty/templates/default/src/main/java/application/api/AssetService.java [hunk 1]
```diff
@@ -76,6 +76,10 @@ public Response getMyAsset(@PathParam("id") String id) throws AssetNotFoundExcep
     @Tag(name = "MyAssets")
     public Response updateMyAsset(@PathParam("id") String id, MyAsset asset) throws IdentityException, GatewayException {
         LOGGER.info("Asset : " + asset.toString());
+        // Give preference to the value passed in the path rather than a possible value in the payload. 
+        if(!asset.getMyAssetId().equals(id)){
+            asset.setMyAssetId(id);
```
Comment (chilanti) [inline incubator/blockchain-client-openliberty/templates/default/src/main/java/application/api/AssetService.java:81]: I am wondering if we should throw an exception if the two IDs do not match...

**Review by chilanti — COMMENTED** [review]
I have one minor comment.

File: incubator/blockchain-client-openliberty/templates/default/src/main/java/application/api/AssetService.java (hunk 1 above)
Comment (rykal-nate) [reply incubator/blockchain-client-openliberty/templates/default/src/main/java/application/api/AssetService.java:81]: Great suggestion.

**Review by chilanti — APPROVED** [review]
LGTM

**Pull request merged**